.github/workflows/build.yml

```diff
@@ -66,7 +63,7 @@ jobs:
                 run: composer install --no-interaction
 
             -   name: Composer Require Checker
-                run: composer-require-checker
+                run: composer check-platform-reqs
```

Comment:
I see error
<img width="914" alt="Снимок экрана 2022-04-15 в 14 09 12" src="https://user-images.githubusercontent.com/1117351/163564165-36df3549-7af2-4233-9d32-06c021a82634.png">

Reply:
https://github.com/PNixx/amp-elasticsearch/runs/5978975713?check_suite_focus=true

Reply:
It's because of this other error: https://github.com/PNixx/amp-elasticsearch/runs/5978975713?check_suite_focus=true#step:6:15

And that error it's because for PHP 7.3 this workflow is still using Composer v1.

There's no reason to use Composer v1. We can use Composer v2 even with PHP 7.3.